### DISCUSSION TITLE
Milliseconds shown in the terminal after build

### DISCUSSION THREAD

**Discussion by abedshaaban** [vercel/next.js#45654] [Help] [answered]
What are the Milliseconds shown in the terminal after the build in the terminal, and what does each color represent (green, yellow, red) ?

![image](https://user-images.githubusercontent.com/72554214/217247906-2f984e02-6df2-46c8-8d48-c51d71041e18.png)

thx in advance :)

**Comment by ciruz** [comment, marked as answer]
According to the code, page size and build duration:

https://github.com/vercel/next.js/blob/12e4175475a1e23b22ab1b6c7e0863410c0276b1/packages/next/src/build/utils.ts#L329

the coloring scheme:

```javascript
  const getPrettySize = (_size: number): string => {
    const size = prettyBytes(_size)
    // green for 0-130kb
    if (_size < 130 * 1000) return chalk.green(size)
    // yellow for 130-170kb
    if (_size < 170 * 1000) return chalk.yellow(size)
    // red for >= 170kb
    return chalk.red.bold(size)
  }

  const MIN_DURATION = 300
  const getPrettyDuration = (_duration: number): string => {
    const duration = `${_duration} ms`
    // green for 300-1000ms
    if (_duration < 1000) return chalk.green(duration)
    // yellow for 1000-2000ms
    if (_duration < 2000) return chalk.yellow(duration)
    // red for >= 2000ms
    return chalk.red.bold(duration)
  }
```

**Reply by abedshaaban** [reply]
thanks for clarifying

**Reply by abedshaaban** [reply]
should i be worried if it is red ? as you can see I have some that are 4332ms and others that are more.
